test/controllers/application_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```

Comment:
So we actually use Rspec for all of our testing. Rspec and Capybara for the more view layer stuff. Take a look at this: https://github.com/learn-co-curriculum/formal-affair-rails/blob/399de77968a7adae2ff1212f95e70ed5028e2895/spec/concrete_form_spec.rb for an idea of what our specs look like